eclair-core/src/main/scala/fr/acinq/eclair/io/Authenticator.scala

```diff
@@ -43,6 +44,7 @@ class Authenticator(nodeParams: NodeParams) extends Actor with DiagnosticActorLo
   def ready(switchboard: ActorRef, authenticating: Map[ActorRef, PendingAuth]): Receive = {
     case pending@PendingAuth(connection, remoteNodeId_opt, address, _) =>
       log.debug(s"authenticating connection to ${address.getHostString}:${address.getPort} (pending=${authenticating.size} handlers=${context.children.size})")
+      Kamon.counter("peers.connecting.count").withTag("state", "authenticating").increment()
```

Comment:
It might be useful to centralize counters for each component (like the `Metrics` object you created for `Peer`).
For example in Authenticator's companion object, define `private val connectingPeersCounter = Kamon.counter("peers.connecting.count")`.
This is usually easier for future maintenance.